frontend/src/scenes/onboarding/sdks/shared-snippets.tsx

```diff
@@ -31,7 +31,7 @@ export function PersonModeEventPropertyInstructions(): JSX.Element {
                 If you want to disable person profile processing for certain events, send the event with the following
                 property:
             </p>
-            <CodeSnippet>"$process_person_profile": false</CodeSnippet>.
+            <CodeSnippet>"$process_person_profile": false</CodeSnippet>
```

Comment:
<img width="503" height="249" alt="Screenshot 2025-07-21 at 14 43 25" src="https://github.com/user-attachments/assets/7804c11f-eea8-408a-afd7-5bf2535962b3" />
causing the dot at the end without any context